core-bundle/src/Resources/contao/library/Contao/InsertTags.php

```diff
@@ -765,6 +766,43 @@ protected function doReplace($strBuffer, $blnCache)
 					break;
 
 				// Images
+				case 'figure':
+					// Expected format: figure::<from>[?<key>=<value>,[&<key>=<value>]*][::<template>]
```

Comment:
Why is `template` not one of the key/value options?

Reply:
The idea was to keep it in line with the definitions of the `Template#figure()` and `contao_figure` Twig functions which accept the template as last parameter.

As we're not respecting this for the `size` as individual argument either, we could as well allow `template=<template>' in the query string. :thinking:

Reply:
Yes please. 😊

Reply:
Changed in 77119d19edb103f1e2c141a3a1d93f9a66b07806.